data/registers/hsem_wl5.yaml

```diff
@@ -0,0 +1,269 @@
+---
+block/HSEM:
+  description: Hardware semaphore
+  items:
+    - name: R
+      description: HSEM register HSEM_R%s HSEM_R31
+      array:
+        len: 16
+        stride: 4
+      byte_offset: 0
+      fieldset: R
+    - name: RLR
+      description: HSEM Read lock register
+      array:
+        len: 16
+        stride: 4
+      byte_offset: 128
+      access: Read
+      fieldset: RLR
+    - name: IER
+      description: HSEM Interrupt enable register
+      byte_offset: 256
+      fieldset: IER
+      array:
+        offsets:
+          - 0
+          - 16
+    - name: ICR
+      description: HSEM Interrupt clear register
+      byte_offset: 260
+      fieldset: ICR
+      array:
+        offsets:
+          - 0
+          - 16
+    - name: ISR
+      description: HSEM Interrupt status register
+      byte_offset: 264
+      access: Read
+      fieldset: ISR
+      array:
+        offsets:
+          - 0
+          - 16
+    - name: MISR
+      description: HSEM Masked interrupt status register
+      byte_offset: 268
+      access: Read
+      fieldset: MISR
+      array:
+        offsets:
+          - 0
+          - 16
+    - name: CR
+      description: HSEM Clear register
+      byte_offset: 320
+      access: Write
+      fieldset: CR
+    - name: KEYR
+      description: HSEM Interrupt clear register
+      byte_offset: 324
+      fieldset: KEYR
+fieldset/ICR:
+  description: HSEM Interrupt clear register
+  fields:
+    - name: ISC
+      description: Interrupt(N) semaphore n clear bit
+      bit_offset: 0
+      bit_size: 1
+      enum_read: C1ICR_ISC0R
```

Comment:
enum_read/enum_write is removed in the latest chiptool, can you remove them?

if we want to keep the enum it's just `enum` now. anyway, the enums here are "0 = not pending, 1 = pending", or "write 1 to clear", which are enums we typically remove.